packaging/windows/build_pypi_pkg_download_proxy/setup.py

```diff
@@ -0,0 +1,79 @@
+import sys
+import subprocess
+
+import setuptools.command.install
+from setuptools import find_packages, setup
+import wheel.bdist_wheel
+
+pkg_name = "torch"
+pkg_ver = "{{GENERATE_TORCH_PKG_VER}}"
+torch_download_url = "https://download.pytorch.org/whl/torch_stable.html"
+
+python_min_version = (3, 6, 1)
+python_min_version_str = '.'.join((str(num) for num in python_min_version))
+
+install_requires = [
+    'wheel',
+    'numpy',
+    'future',
+    'typing_extensions',
+    'dataclasses; python_version < "3.7"'
```

Comment:
Why should the wrapper package depend on anything but pip? (and 'pip' depends on 'wheel')
```suggestion
   'pip',
```